lib/podio/models/invoice.rb

```diff
@@ -14,6 +14,10 @@ class << self
     def find(invoice_id)
       member Podio.connection.get("/invoice/#{invoice_id}").body
     end
+    
+    def find_vers2(invoice_id, contract_id)
```

Comment:
Please rename the method to find_v2

Reply:
I named it as vers2 , just to be in sync with the other method names in the same file
https://github.com/podio/podio-rb/blob/befdf56009a73a33563340e6edbd5afc815cf6e6/lib/podio/models/invoice.rb#L26
and
https://github.com/podio/podio-rb/blob/befdf56009a73a33563340e6edbd5afc815cf6e6/lib/podio/models/invoice.rb#L51

Should I still go with find_v2.. pls let me know?

Reply:
I think go with _v2, we can rename existing methods later